setup.py

```diff
@@ -117,6 +117,10 @@
         'dojson.contrib.marc21': [
             'cernopendata = cernopendata.rules',
         ],
+       'flask.commands': [
```

Comment:
Not sure if we actually need this, we can only keep ```console.scripts``` one and register everything in the general cli like here https://github.com/zenodo/zenodo/blob/master/zenodo/cli.py

Reply:
Yup, it would be good indeed. Let's change this later, also in terms of fixtures we need to complete them, e.g. only terms are plugged in, not the data policies or the news articles yet.  Kept it the same for the merge...

Reply:
Alright cool ! 👍

Reply:
(BTW I'll repush to fix the Travis CI light)